.github/actions/download-all-weekly-artifacts/action.yaml

```diff
@@ -29,14 +29,42 @@ runs:
         binary-artifact-name: ${{ inputs.prefix }}gcc-linux-rv64gcv_zve64d-lp64d-${{ inputs.gcchash }}-multilib
         github-token: ${{ inputs.token }}
 
+    # Download rv64 zvl builds.
+
     - name: Download linux rv64-1024b
       uses: ./.github/actions/common/download-comparison-artifacts
-      if: ${{ inputs.prefix == 'rv64_zvl_' }}
+      if: ${{ inputs.prefix == 'rv64_zvl_' || inputs.prefix == 'rv64_zvl_lmul2_' }}
```

Comment:
I'm assuming this is future-proofing since afaict we don't have lmul=2 runs currently?

Reply:
We have started doing rv64_zvl_lmul2 runs as of 16675b165c696551e16fa8a15442aade8d85ef12